src/Network/Ethereum/Web3/Types/Types.purs

```diff
@@ -462,6 +462,7 @@ forkWeb3' web3Action = do
 -- * Filters
 --------------------------------------------------------------------------------
 -- | Low-level event filter data structure
+newtype Filter :: forall k. k -> Type
```

Comment:
is this actually required by the compiler? I don't think this is something that is really needed for simple polymorphic datatypes like `Filter`